components/build/jvm-build-service/operator-images.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hacbs-jvm-operator
+spec:
+  template:
+    spec:
+      containers:
+        - name: hacbs-jvm-operator
+          env:
+            - name: JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE
+              value: "quay.io/redhat-appstudio/hacbs-jvm-build-request-processor:3dcd8603061c7ab2cedfe128c80cac9726721a74"
+            - name: JVM_BUILD_SERVICE_SIDECAR_IMAGE
```

Comment:
@gabemontero I have added this to make your https://github.com/redhat-appstudio/jvm-build-service/pull/118 PR work. This is now set in two places, so it should work both with an without that PR.

Reply:
awesome / thanks @stuartwdouglas saves me a step